addon/metrics-adapters/new-relic.js

```diff
@@ -0,0 +1,62 @@
+import BaseAdapter from 'ember-metrics/metrics-adapters/base';
+import { inject as service } from '@ember/service';
+import { get, set } from '@ember/object';
+import { assert } from '@ember/debug';
+
+const CONFIG_DEFAULTS = {
+  beacon: 'bam.nr-data.net',
+  errorBeacon: 'bam.nr-data.net',
+  sa: 1
+};
+
+// This will be replaced with the real snippet at build-time
+export const SNIPPET = `NEW_RELIC_SNIPPET`;
+
+export default BaseAdapter.extend({
+  newRelic: service(),
+
+  _scriptTag: undefined,
+
+  toStringExtension() {
+    return 'new-relic';
+  },
+
+  init() {
+    const info = Object.assign({}, CONFIG_DEFAULTS, get(this, 'config'));
+    const { licenseKey, applicationID } = info;
+
+    assert(
+      `[ember-metrics] You must pass a valid \`licenseKey\` to the ${this.toString()} adapter`,
+      licenseKey
+    );
+    assert(
+      `[ember-metrics] You must pass a valid \`applicationID\` to the ${this.toString()} adapter`,
+      applicationID
+    );
+
+    const scriptTag = document.createElement('script');
```

Comment:
Basically, instead of all the complicated build-time configuration that was necessary before, we just use Broccoli to find-and-replace the `SNIPPET` variable's value with the contents of the snippet itself (which we can keep as a separate file for easy updating) and initialize the script here.

It feels kind of weird to me that you would inject the script this way, but this is how `ember-metrics` adapters are supposed to operate, so I'm trusting that.

Likewise, `willDestroy` down at the bottom removes the script tag and nuked the global that New Relic provides.